rosidl_typesupport_introspection_c/resource/srv__type_support.c.template

```diff
@@ -20,35 +20,60 @@ function_prefix = '%s__srv__rosidl_typesupport_introspection_c' % spec.pkg_name
 #ifndef @(header_guard_variable)
 #define @(header_guard_variable)
 
-#include <rosidl_generator_c/service_type_support.h>
+#include <rosidl_generator_c/message_type_support.h>
```

Comment:
This file later uses `rosidl_service_type_support_t` which is defined in `service_type_support.h`?